oaknational/Oak-Web-Application
# fix: homepage classroom teacher links to open in new tab

## Description

- open links in a new tab

## Issue(s)

Fixes #

## How to test

1. Go to {cloud link}
2. Click on _______
3. Should open in new tab

## Checklist
